### DISCUSSION TITLE
Using Gitbucket Auth instead of Local User Auth.

### DISCUSSION THREAD

**Discussion by Sudro** [concourse/concourse#9015] [Help & Support]
Hello. I have a Docker Compose that describes Concourse CI. So far, I have described the authentication logic of the local auth user. But when adding new users (when changing the CONCOURSE_ADD_LOCAL_USER parameter), I have to restart the Concourse CI service, which is described in docker compose, every time to apply the changes. 
I found authentication settings using various methods in the documentation ( https://concourse-ci.org/configuring-auth.html ). But I am working with GitBucket, which, as far as I understand, does not support these methods (creating an OAuth application).
Tell me, please, how can I solve my problem?

**Comment by taylorsilva** [comment]
I'm not familiar with GitBucket (https://gitbucket.github.io/). You need to find out if GitBucket supports being an oAuth or OIDC provider (not consumer, like Concourse). It sounds like your research has turned up that it doesn't. You might want to check-in with the GitBucket community to confirm though.

An alternative would be to use some platform for auth with Concourse. For example, you could make a GitHub organization that you use purely for authentication with Concourse. You could do a similar setup with any other platform that you could use as an oAuth or OIDC provider. That's the only workaround I can think of for your situation.

**Reply by Sudro** [reply]
Thank you!
